patch.md

```diff
@@ -0,0 +1,37 @@
+# General rules for the Patch operation
+[Back to the index](README.md)
+
+The PATCH method expects a JSON body according to the [JSON Patch specification RFC6902](https://tools.ietf.org/html/rfc6902)
+
+Each successful Patch operation will return a **HTTP 200 CODE** with the new state of the patched resource as body similar to what is returned for a GET request. 
+
+### Add
+The add operation is expected to be supported by all the endpoints that implement the Patch method for all the json path not flagged as **READ-ONLY**.
+
+Please note that according to the [JSON Patch specification RFC6902](https://tools.ietf.org/html/rfc6902) a subsequent add operation on an already set attribute have the effect to replace the previous value.
+
+This use of the add operation to replace a value could be counter intuitive but it is done according to the [RFC section 4.1](https://tools.ietf.org/html/rfc6902#section-4.1)
+> If the target location specifies an object member that does exist, that member's value is replaced.
```

Comment:
I don't understand your first example. Which member value we will need to support? are you talking about a not repeatable metadata? I suggest to have all metadata exposed as repeatable because you can always hit the situation of mess data in the database and it will be easier to report all the values to the client instead to throw exceptions or hide information.

> I've you execute the last command 3 times, you will have added 3 times the same author. The item will thus contains three authors.
this is the expected behavior from the spec